components/dash-core-components/src/utils/computeSliderMarkers.js

```diff
@@ -144,15 +166,18 @@ export const sanitizeMarks = ({min, max, marks, step}) => {
         return undefined;
     }
 
+    const [min_mark, max_mark] = Object.values(setUndefined(min, max, marks));
```

Comment:
```suggestion
    const {min_mark, max_mark} = setUndefined(min, max, marks);
```
Does the same thing right? Except the previous way is dependent on `Object.values` always putting `min_mark` first, which is implementation-dependent (implementation of JS itself and our implementation of `setUndefined`)